pakyow-presenter/lib/pakyow/presenter/presentable.rb

```diff
@@ -8,13 +8,27 @@ def initialize(*args)
         super
       end
 
-      def presentable(*args)
-        presentables.concat(args).uniq!
+      def presentable(name, value = nil)
+        args = {
+          name: name,
+          value: value
+        }
+
+        args[:block] = Proc.new if block_given?
```

Comment:
I see why you did this, but I was hoping to just yield. The only way we could do that is if we called the block immediately instead of when we setup the presentation context. But then it would behave differently than a method call. Unsure what approach to take here; perhaps this is worth a discussion.